### ISSUE TITLE
Competitive Coding

### ISSUE THREAD

**Issue by Akshitaag** [GirlScriptSummerOfCode/Projects#14]
Project: Competitive Coding

Description:It contains some useful codes, techniques, algorithms, data structure implementations and problem solutions helpful in Competitive Coding. It can help students prepare for competitive coding contests and also help them in learning new data structures and algorithms.	C++	Should help students new to open source become familiar with it.

Tech Stack: C++

Admin: Sahil Bansal 
 Link : https://github.com/sahilbansal17/Competitive_Coding
Mentors: Vikas gola ,Aditya Agarwal, mayank408, Nakul Sabharwal, Ramit Sahwney

**Comment by ashubly25** [comment]
I would like to work on this.

**Comment by ashubly25** [comment]
@sahilbansal17 I would like to work on this.

**Comment by sahilbansal17** [comment]
@ashubly25 Sure, you should join the GSSoC Slack channel for this project.

**Comment by ashubly25** [comment]
Yeah, I am already there. Do you use gitter or a dedicated slack channel for communication for project? GSSoc would be clutter free

**Comment by sahilbansal17** [comment]
@ashubly25 No, we can only communicate using GSSoC Slack channel.

**Comment by ashubly25** [comment]
Yeah sure.

**Comment by dhairyabaweja** [comment]
I want to contribute to this project.

**Comment by vaarigupta** [comment]
I am willing to contribute in this repo

**Comment by hyp3r5pace** [comment]
I am trying to find your slack channel but i can't. Can you provide me the link?
